### DISCUSSION TITLE
Error converting to newer verion of asreview lab

### DISCUSSION THREAD

**Discussion by taviaevans** [asreview/asreview#1407] [Troubleshooting (report errors in the "Issues" tracker)]
Hi, I have upgraded to the newer asReview lab and it asks me to upgrade the projects however when I go to upgrade it states: "An error occurred during conversion of state from 0 to 1. Please try again. " 

Any advice on how to overcome this and upgrade my project so I can use it in the new version?

Thanks!

**Comment by J535D165** [comment]
Sorry for inconvenience. Would you mind sending us the project folder?

Can you find a folder named ".asreview" under This PC or on MacOS in the Home directory? Is there a subfolder that matches your project name more or less? Can you send us that project folder to asreview@uu.nl. Let me know if you need help!

**Comment by PeterLombaers** [comment]
Hi! I'm not completely sure what was the cause of the problem. The project file was corrupted, but all the data was still in there. I manually reconstructed the file, so now it should be working fine with ASReview version 1. You should have the fixed file in your email.
